src/renderer/pages/utils.ts

```
@@ -0,0 +1,19 @@
+export const insertSpeakers = (data: MulmoScript) => {
```

ESLint

[error] 'MulmoScript' is not defined.

(no-undef)